app/helpers/validation_request_helper.rb

```diff
@@ -38,4 +38,14 @@ def display_request_status(validation_request)
       "green"
     end
   end
+
+  def request_sent_at(validation_request)
```

Comment:
I know it's not in the designs, but wondering if we should have a "Not sent" or similar to draw it to the officer's attention if they have created requests without also invalidating?

Reply:
Its a good point - I'll merge this one and ask Tom to check during UAT / UX to see if the officers need that extra reinforcement - he might even prefer an alert box at the top or so